src/partials/SidebarRight.svelte

```diff
@@ -1,11 +1,15 @@
 <script>
+  import { stores } from '@sapper/app';
   import { createEventDispatcher } from 'svelte';
   import { Search, Select, SelectItem } from "carbon-components-svelte";
 
   export let display = ['events', 'posts'];
   export let events = [];
   export let posts = [];
   export let filters = [];
+  export let anchors = [];
```

Comment:
So any sub-headings will show up in the right sidebar? I like that idea.

Reply:
Thanks @mukhtyar and yes, that was my thinking. It could be helpful for longer articles with many headings > h1 but we might want to disable it for articles that have only a single h2 heading.